src/prettier-comments/common/util.js

```diff
@@ -430,11 +430,7 @@ function getIndentSize(value, tabWidth) {
   );
 }
 
-function printString(raw, options, isDirectiveLiteral) {
-  // `rawContent` is the string exactly like it appeared in the input source
-  // code, without its enclosing quotes.
-  const rawContent = raw.slice(1, -1);
-
+function printString(rawContent, options) {
```

Comment:
I modified this function to make it more useful for us. It wasn't being used anywhere, so it shouldn't be a problem.